pyracf/resource/resource_admin.py

```diff
@@ -589,3 +718,43 @@ def _format_profile(self, result: dict) -> None:
 
         del result["securityResult"]["resource"]["commands"][0]["messages"]
         result["securityResult"]["resource"]["commands"][0]["profiles"] = profiles
+
+    def __validate_access_level(
+        self,
+        success: Union[str, None] = None,
+        failure: Union[str, None] = None,
+        all: Union[str, None] = None,
+    ):
+        access_levels = ["alter", "control", "read", "update"]
```

Comment:
Make this a tuple:

```python
access_levels = ("alter", "control", "read", "update")
```

Reply:
May also make sense to rename this to `valid_access_levels`.

Reply:
sure, changes made.